common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4756,6 +4756,13 @@ public static enum ConfVars {
         "Class to use for calculating available slots during split generation"),
     HIVE_TEZ_GENERATE_CONSISTENT_SPLITS("hive.tez.input.generate.consistent.splits", true,
         "Whether to generate consistent split locations when generating splits in the AM"),
+    HIVE_TEZ_SPLIT_FS_SERIALIZATION_THRESHOLD("hive.tez.split.fs.serialization.threshold", 268435456,
```

Comment:
should it be `hive.tez.input.fs.serialization.threshold` similar to HIVE_TEZ_GENERATE_CONSISTENT_SPLITS("hive.tez.input.generate.consistent.splits"?

Reply:
make sense, let me change